test/avocado/selenium-login.py

```diff
@@ -134,23 +174,24 @@ def login(self, tmpuser=user, tmppasswd=passwd):
         elem = self.wait_id('login-password-input')
         elem.clear()
         elem.send_keys(tmppasswd)
-        self.wait_id("login-button").click()
+        self.click(self.wait_id("login-button", cond=clickable))
         return elem
 
     def logout(self):
-        elem = self.wait_id('navbar-dropdown')
-        elem.click()
-        elem = self.wait_id('go-logout')
-        elem.click()
+        elem = self.wait_id('navbar-dropdown', cond=clickable)
```

Comment:
If the element isn't reused, you could chain it like [here](https://github.com/cockpit-project/cockpit/pull/3457/files#diff-dea380264985bf17f7625dcb4b42b52fR177)